jujugui/static/gui/src/app/init.js

```diff
@@ -1488,7 +1488,7 @@ class GUIApp {
         deployTarget: null
```

Comment:
So this branch updates the method but leaves the state as deployTarget? Are you planning on doing a follow-up to update state and then keep support for the old/new url scheme?